pyuvdata/uvbeam.py

```diff
@@ -900,14 +936,126 @@ def get_lambda(real_lut, imag_lut=None):
 
         return interp_data, interp_basis_vector
 
-    def interp(self, az_array=None, za_array=None, freq_array=None, reuse_spline=False):
+    def _interp_healpix_bilinear(self, az_array, za_array, freq_array, freq_interp_kind='linear',
+                                 polarizations=None):
+        """
+        Simple bi-linear interpolation wrapper for healpix.
+
+        Args:
+            az_array: azimuth angles to interpolate to [radians]
+            za_array: zenith angles to interpolate to [radians]
+            freq_array: frequency values to interpolate to [Hz]
+            freq_interp_kind: str, interpolation method across frequency. See scipy.interpolate.interp1d for details.
```

Comment:
shouldn't the `freq_interp_tol` show up here in the same way it does in `_interp_az_za_rect_spline`?